examples/complete/main.tf

```diff
@@ -91,16 +91,18 @@ module "ecs" {
             condition     = "START"
           }]
 
-          enable_cloudwatch_logging = false
+          enable_cloudwatch_logging              = true
+          create_cloudwatch_log_group            = true
+          cloudwatch_log_group_name              = "/aws/ecs/${local.name}/${local.container_name}"
+          cloudwatch_log_group_retention_in_days = 7
+
           log_configuration = {
-            logDriver = "awsfirelens"
```

Comment:
we shouldn't remove/overwrite this - we have two other examples where we can implement this

Reply:
got it!
Do you not want to touch the `complete` example?

Reply:
moved to the `autoscaling` example

Reply:
>  Do you not want to touch the complete example?

we shouldn't delete the only example of a select configuration in lieu of another if we have other examples where it can be demonstrated without removing anything else